htdocs/core/lib/product.lib.php

```diff
@@ -360,9 +360,9 @@ function product_lot_admin_prepare_head()
  */
 function show_stats_for_company($product, $socid)
 {
-	global $conf, $langs, $user, $db;
+	global $conf, $langs, $user, $db, $hookmanager;
 	$form = new Form($db);
-
+	$hookmanager->initHooks(array('productstats'));
```

Comment:
the inithook must be done by the parent page (the main php file that include all other). So a init should have already be done when we enter the show_stats_for_company(), so i removed this line.